crates/client/eth/src/l1_messaging.rs

```diff
@@ -150,14 +155,14 @@ async fn process_l1_message(
     let blockifier_transaction: BlockifierL1HandlerTransaction =
```

Comment:
is there a difference between this, and blockifier Transaction::from_api with the starknet api transaction?

Reply:
nope there isn't, refactoring